app/views/templates/base.php

```diff
@@ -112,30 +122,29 @@
 
   <div id="footer">
     <p>Transvision is a tool provided by the French Mozilla community, <a href="https://www.mozfr.org" title="Home of MozFR, the French Mozilla Community" hreflang="fr">MozFR</a>.</p>
-    <?php echo $last_update; ?>
+    <?= $last_update ?>
```

Comment:
I don't have a strong preference, I am thinking that spaces makes reading the variable name maybe more easily maybe? I let you choose on this one and I'll update the file for whatever choice we do.